SonataPageBundle.php

```diff
@@ -59,10 +59,18 @@ public function boot()
         }
 
         call_user_func(array($class, 'setSlugifyMethod'), function ($text) use ($container) {
+            // NEXT_MAJOR: remove this check
             if ($container->hasParameter('sonata.page.slugify_service')) {
                 $id = $container->getParameter('sonata.page.slugify_service');
             } else {
-                $id = 'sonata.core.slugify.native'; // default BC value, you should use sonata.core.slugify.cocur
+                @trigger_error(
+                    'The "sonata.core.slugify.native" service is deprecated since 3.0, to be removed in 4.0. '.
```

Comment:
According to https://github.com/sonata-project/SonataPageBundle/commit/a4ce19251b70618aa3334455891998e1baa81fa8, the BC commit is here since `2.3.9`.

I think this version should be put on the message.